server/utils/CacheHelper.ts

```diff
@@ -1,32 +1,30 @@
 import { Day } from "@shared/utils/time";
 import Logger from "@server/logging/Logger";
 import Redis from "@server/storage/redis";
-import { Unfurl, UnfurlSignature } from "@server/types";
 
 /**
  * A Helper class for server-side cache management
  */
 export class CacheHelper {
+  // Default expiry time for cache data in ms
   private static defaultDataExpiry = Day;
 
   /**
    * Given a key, gets the data from cache store
    *
    * @param key Key against which data will be accessed
    */
-  public static async getData(key: string): ReturnType<UnfurlSignature> {
```

Comment:
Weird that this helper had a bunch of Unfurl types @apoorv-mishra

Reply:
Yeah, doesn't quite make sense for a generic(hopefully) helper's methods. I don't recall a specific reason for this though. Perhaps, just an oversight.